packages/kyt-core/package.json

```diff
@@ -46,7 +46,7 @@
     "jest": "16.0.1",
     "kyt-utils": "0.1.0",
     "lodash.clonedeep": "4.5.0",
-    "node-sass": "3.10.1",
+    "node-sass": "^4.5.0",
```

Comment:
We've been pinning assets by removing the semver ranges. Can you remove the `^` 🙏